site/content/resources/_index.html

```diff
@@ -0,0 +1,166 @@
+---
+title: "Resources"
```

Comment:
@amymanion if there's time, I would refactor this to use [jekyll collections](https://jekyllrb.com/docs/collections/) and liquid templating instead, so that in the future authors can simply add a new resource in markdown, instead of having to touch the HTML. that way it is less brittle and prone to error, and easier for others to update.  you would then loop over each item in the collection within partials and authors then need only add a new markdown item in the collection for new videos, podcasts, or blogs